chromium_src/ios/chrome/browser/ios_chrome_field_trials.cc

```diff
@@ -0,0 +1,45 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "ios/chrome/browser/ios_chrome_field_trials.h"
+
+#include "base/files/file_path.h"
+#include "base/files/file_util.h"
+#include "base/path_service.h"
+#include "base/task/post_task.h"
+#include "base/task/thread_pool.h"
+#include "components/metrics/persistent_histograms.h"
```

Comment:
this header isn't used and I think you're missing a header for base::FeatureList

Reply:
I ran `npm run gn_check ios_Release` and it printed `Header dependency check OK` in green.
I did remove the unused header though.

Reply:
`base::FeatureList` gets included via `ios/chrome/browser/ios_chrome_field_trials.h` 👍 so good to go there. We should be able to just remove line 13

Reply:
@bsclifton you should not depend on indirect includes https://github.com/include-what-you-use/include-what-you-use/blob/master/docs/WhyIWYU.md#allow-refactoring

Reply:
Resolved with https://github.com/brave/brave-core/commit/1d699fa66050f390cb4b6a646f690a9f09c116cf - thanks for the heads up @bridiver 😄